local-cli/rnpm/windows/package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "rnpm-plugin-windows",
-  "version": "0.2.8",
+  "version": "0.2.9",
   "description": "rnpm plugin that generates a Windows template project",
```

Comment:
Do we need to bump this?

Reply:
I guess not which is why I initially bumped only the patch number (major.minor.patch) as there was no breaking changes. I do think we should publish the changes to npm so it should be at least a patch bump.